docker/validator-dynamic/build.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/sh
 # Copyright (c) The Libra Core Contributors
 # SPDX-License-Identifier: Apache-2.0
-true
+set -e
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
+
+PROXY=""
+if [ "$https_proxy" ]; then
+	    PROXY=" --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy"
+fi
+
+docker build -f $DIR/Dockerfile $DIR/../.. --tag libra_validator_dynamic --build-arg GIT_REV="$(git rev-parse HEAD)" --build-arg GIT_UPSTREAM="$(git merge-base HEAD origin/master)" --build-arg BUILD_DATE="$(date -u +'%Y-%m-%dT%H:%M:%SZ')" $PROXY "$@"
```

Comment:
Yikes, CI failed https://circleci.com/gh/libra/libra/54579.  

It looks like we need to build validator first, reason being in docker/validator-dynamic/Dockerfile
```
 FROM libra_e2e:latest as validator_with_config
```